File: Examples/Scala/ConditionalTest.scala
object ConditionalTest {

  def main(args: Array[String]) {
    var x = 0
    var i = 0
    while (i < 100) {
      println("Idx: " + i + ";Val: " + x)
      x += 1
      i += 1
    }
  }
}

File: Examples/Java/ConditionalTest.java
/* Name of the class has to be "Main" only if the class is public. */
class ConditionalTest
{
	public static void main (String[] args) {
    int x = 0;
    for (int i = 0; i < 100; i++)
			System.out.println("Idx: " + i + ";Val: " + x++);
	}
}
